### ISSUE TITLE
The accuracy following esc50 Recipe is very low

### ISSUE THREAD

**Issue by nikhilbyte** [YuanGongND/ast#17]
There must be some mistake from my side. Can someone help me identify it?<img width="1164" alt="Screenshot 2021-08-29 at 5 42 13 PM" src="https://user-images.githubusercontent.com/62005061/131255696-248aa192-447b-4b85-a760-6ef3bb96b798.png">
This is how I'm training:
`!python -W ignore /content/ast/src/run.py --model ast --dataset esc50 \
  --data-train /content/data/datafiles/esc_train_data_1.json --data-val /content/data/datafiles/esc_eval_data_1.json --exp-dir /content/expdir/fold1 \
  --label-csv /content/ast/egs/esc50/data/esc_class_labels_indices.csv --n_class 50 \
  --lr 1e-5 --n-epochs 25 --batch-size 12 --save_model False \
  --freqm 24 --timem 96 --mixup 8 --bal None \
  --tstride 10 --fstride 10 --imagenet_pretrain True --audioset_pretrain True`

**Comment by YuanGongND** [comment]
Hi there,

What I noticed is that mixup should be in the range of [0-1], so 8 is out of range. You also used a smaller batch size, which might also have some impact (in that case, you need to use a smaller learning rate). The learning rate scheduler is based on the epochs, so when you decrease the batch size, you should also change the learning rate scheduler. There might be some other things, the model is not trained at all.

Anyway, the easiest way to reproduce the result is using the parameters in the [recipe](https://github.com/YuanGongND/ast/blob/master/egs/esc50/run_esc.sh), if you don't have enough GPU memory (we use 4 * 12GB GPUs), it is better to set the ``tstride`` and ``fstride = 16`` (which saves a lot of GPU memory) and keep other things unchanged.  You won't get the same result as in the paper for larger strides, but it should be similar.

-Yuan

**Comment by YuanGongND** [comment]
Also, are you using our ``prep_data.py``? It converts the audios to 16kHz sampling rate, the original is 44.1kHz.

**Comment by nikhilbyte** [comment]
Hello Yuan, Thanks for the quick reply.

> Also, are you using our `prep_data.py`? It converts the audios to 16kHz sampling rate, the original is 44.1kHz.

Yes I used the prep_data.py without changing anything.

Changing the mixup to the default value helped getting the accuracy higher. 
Thanks a lot.

**Comment by YuanGongND** [comment]
Great to know. Also you can compare your log with our [log file](https://github.com/YuanGongND/ast/blob/master/egs/esc50/exp/test-esc50-f10-t10-pTrue-b48-lr1e-5/log_22108.txt) to see if everything is going well.

**Comment by nikhilbyte** [comment]
Yes, I did compare it with the log files you provided. Thanks
